PowerEditor/src/WinControls/FunctionList/functionParser.cpp

```diff
@@ -598,14 +598,14 @@ void FunctionParser::getInvertZones(vector< pair<int, int> > &  destZones, vecto
 {
 	if (sourceZones.size() == 0)
 	{
-		destZones.push_back(pair<int, int>(begin, end));

```

Comment:
cleaning up this area seems to be a different story, just slightly related to x64
so I would consider that as a separate PR for cleaning up the plenty of x64 compiler warnings regarding:

warning C4267|C4244: 'Z' : conversion from 'X' to 'Y', possible loss of data